src/modules/utils.js

```diff
@@ -0,0 +1,146 @@
+//import fs
+const fs = require("fs");
+
+const generateHtml = ({ teamName, managers, engineers, interns }) => {
```

Comment:
Depending on whether you refactor in `index.js` as advised, you would prep this like:

`const generateHtml = (teamName, , { managers, engineers, interns }) => { `